whats-new/prime-yield.md

```diff
@@ -33,6 +33,7 @@ Venus Prime encourages user commitment through two unique Prime Tokens:
     * Users need to stake at least 1,000 XVS for 90 days in a row.
     * After these 90 days, users can mint their Prime Token.
     * If a user decides to withdraw XVS and their balance falls below 1000, their Prime Token will be automatically revoked.
+    * Limit to the number of Prime holders is 500 tokens on BSC-chain. [Source](https://app.venus.io/#/governance/proposal/201)
```

Comment:
```suggestion
    * The limit to the number of revocable Prime tokens is 500 on BNB chain. [Source](https://app.venus.io/#/governance/proposal/201). It can be changed with a VIP.
```

Reply:
nice